Typescript errors

```
src/ui/components/ArrowLink/component.ts(15,69): error TS1005: ';' expected.
src/ui/components/Simplabs/component.ts(71,24): error TS2459: Type '{ component: string; }' has no property 'title' and no string index signature.
src/ui/components/Simplabs/component.ts(71,36): error TS2459: Type '{ component: string; }' has no property 'bundle' and no string index signature.
src/ui/components/Simplabs/component.ts(71,44): error TS2459: Type '{ component: string; }' has no property 'parentBundle' and no string index signature.
src/ui/components/Simplabs/component.ts(79,9): error TS2322: Type '{ after: () => void; already: () => void; }' is not assignable to type 'INavigoHooks'.
  Object literal may only specify known properties, and 'already' does not exist in type 'INavigoHooks'.
src/ui/components/Simplabs/component.ts(84,9): error TS2322: Type '(done: any, params: any) => Promise<void>' is not assignable to type '(done: () => void) => void'.
src/ui/components/Simplabs/component.ts(90,24): error TS2339: Property 'Sentry' does not exist on type 'Window'.
src/ui/components/Simplabs/component.ts(91,22): error TS2339: Property 'Sentry' does not exist on type 'Window'.
src/ui/components/Simplabs/component.ts(93,13): error TS2322: Type 'string' is not assignable to type 'Location'.
src/ui/components/Simplabs/component.ts(216,32): error TS2339: Property 'innerText' does not exist on type 'Element'.
src/ui/components/Simplabs/component.ts(224,14): error TS2339: Property 'ga' does not exist on type 'Window'.
src/ui/components/Simplabs/component.ts(225,12): error TS2339: Property 'ga' does not exist on type 'Window'.
src/ui/components/Simplabs/component.ts(226,12): error TS2339: Property 'ga' does not exist on type 'Window'.
```

--- a/config/routes-map.js
+++ b/config/routes-map.js
@@ -1,7 +1,6 @@
 'use strict';
 
 const path = require('path');
-
 const collectPosts = require('../lib/generate-blog/lib/collect-posts');
 
 module.exports = function() {

--- a/src/ui/components/ArrowLink/component.ts
+++ b/src/ui/components/ArrowLink/component.ts
@@ -12,7 +12,7 @@ export default class ArrowLink extends Component {
   get isSimplabs() {
     let href = this.args.href || '';
 
-    return this.isInternal || href.match(/^https?:\/\/simplabs.com/));
+    return this.isInternal || href.match(/^https?:\/\/simplabs.com/);
   }
 
   @tracked

--- a/src/ui/components/Simplabs/component.ts
+++ b/src/ui/components/Simplabs/component.ts
@@ -7,6 +7,24 @@ const COOKIE_BANNER_DELAY: number = 100;
 interface INavigoHooks {
   before?: (done: () => void) => void;
   after?: () => void;
+  already?: () => void;
+}
+
+declare global {
+  // tslint:disable-next-line:interface-name
+  interface Window {
+    Sentry: any;
+    ga: any;
+  }
+}
+
+interface IRoutesMap {
+  [route: string]: {
+    component: string;
+    title: string;
+    bundle: any;
+    parentBundle: any;
+  };
 }
 
 export default class Simplabs extends Component {
@@ -39,7 +57,7 @@ export default class Simplabs extends Component {
       isSSR: false,
       origin: window.location.origin,
       route: window.location.pathname,
-      routesMap: this._readRoutesMap()
+      routesMap: this._readRoutesMap(),
     };
 
     this._setupRouting();
@@ -67,8 +85,8 @@ export default class Simplabs extends Component {
   private _setupRouting() {
     this.router = new Navigo(this.appState.origin);
 
-    Object.keys(this.appState.routesMap).forEach((path) => {
-      let { component, title = '', bundle, parentBundle } = this.appState.routesMap[path];
+    Object.keys(this.appState.routesMap).forEach(path => {
+      let { component, title = '', bundle, parentBundle } = (this.appState.routesMap as IRoutesMap)[path];
       let options: INavigoHooks = {
         after: () => {
           this._setPageTitle(title);
@@ -78,19 +96,20 @@ export default class Simplabs extends Component {
         },
         already: () => {
           window.scrollTo(0, 0);
-        }
+        },
       };
       if (bundle && !this.appState.isSSR) {
-        options.before = async (done, params) => {
+        options.before = async done => {
           try {
             await this._loadBundle(bundle, parentBundle);
             this._registerBundle(bundle);
             done();
-          } catch(e) {
+          } catch (e) {
             if (window.Sentry) {
               window.Sentry.captureException(e);
             }
-            window.location = path;
+
+            window.location.href = path;
           }
         };
       }
@@ -111,7 +130,7 @@ export default class Simplabs extends Component {
       );
     });
 
-    this.router.notFound(() => this.activeComponent = 'Page404');
+    this.router.notFound(() => (this.activeComponent = 'Page404'));
 
     this.router.resolve(this.appState.route);
   }
@@ -211,7 +230,7 @@ export default class Simplabs extends Component {
   }
 
   private _readRoutesMap() {
-    let script = document.querySelector('[data-shoebox-routes]');
+    let script: HTMLElement | null = document.querySelector('[data-shoebox-routes]');
     if (script) {
       return JSON.parse(script.innerText);
     } else {

